index.bs

```diff
@@ -2302,6 +2355,30 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
 Issue: Invoke [=parse summary buckets=] and [=parse summary window operator=]
 from this algorithm.
 
+To <dfn>parse max event states</dfn> from a [=map=] |map|,
+a possibly null 32-bit unsigned integer |attributionScopeLimit|:
+1. If |attributionScopeLimit| is null:
+    1. If |map|["<code>[=source-registration JSON key/max_event_states=]</code>"] [=map/exists=], return an error.
+    1. Return null.
+1. Return the result of running
```

Comment:
Don't we need to check that the value is positive, since the corresponding source field is `Null or a positive unsigned 32-bit integer`?

That said, I'm not sure we need to prevent 0, as it is possible that some user of the API would only care about navigation sources, not event ones, in which case it can guarantee event sources are never used by setting 0 here.

Reply:
it should be covered by the newly added `parse an optional 64-bit positive integer `

Reply:
Right, but why don't we want to allow 0?